nexus/src/app/sagas/project_create.rs

```diff
@@ -51,20 +55,26 @@ impl NexusSaga for SagaProjectCreate {
     }
 
     fn make_saga_dag(
-        _params: &Self::Params,
+        params: &Self::Params,
         mut builder: steno::DagBuilder,
     ) -> Result<steno::Dag, super::SagaInitError> {
         builder.append(project_create_record_action());
-        builder.append(project_create_vpc_params_action());
-
-        let subsaga_builder = steno::DagBuilder::new(steno::SagaName::new(
-            sagas::vpc_create::SagaVpcCreate::NAME,
-        ));
-        builder.append(steno::Node::subsaga(
-            "vpc",
-            sagas::vpc_create::create_dag(subsaga_builder)?,
-            "vpc_create_params",
-        ));
+
+        // Only create default VPC if allowed (i.e., networking is not restricted
+        // or the actor is a Silo Admin)
```

Comment:
```suggestion
        // Only create default VPC if requested.
```

I'd suggest that the question of whether this is set is a policy choice by the caller and this code doesn't need to care about when or why it's set.